instrumentation/instaawssdk/instaawssdk.go

```diff
@@ -58,11 +64,13 @@ func InstrumentSession(sess *session.Session, sensor *instana.Sensor) {
 			}
 		case dynamodb.ServiceName:
 			FinalizeDynamoDBSpan(req)
+		case lambda.ServiceName:
+			FinalizeInvokeLambdaSpan(req)
 		}
 	})
 }
 
-func injectTraceContext(sp opentracing.Span, req *request.Request) {
+func injectTraceContext(logger instana.LeveledLogger, sp opentracing.Span, req *request.Request) {
```

Comment:
It's a matter of taste, but can we put auxiliary dependency to the end of parameter list?

Reply:
changed in next commit